test/controllers/active_storage/blobs_controller_test.rb

```diff
@@ -40,9 +40,7 @@ class ActiveStorage::BlobsControllerTest < ActionDispatch::IntegrationTest
         subject
       end
 
-      it 'flashes an unauthorized message' do
```

Comment:
To fix, though I don't think this actually worked before, but was ignored due to missing translations.

Reply:
Removed due to being not needed.